earth2mip/initial_conditions/ifs.py

```diff
@@ -29,8 +30,8 @@
 
 
 def _get_filename(time: datetime.datetime, lead_time: str):
```

Comment:
Could you add a doctring to this? In it, could you add a note on why the file format is what it is, since it is very specific? Also since we're here, it would be nice to add a return type.

Reply:
done - thanks for the suggestions!